src/main/java/network/brightspots/rcv/AuditableFile.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * RCTab
+ * Copyright (c) 2017-2023 Bright Spots Developers.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Purpose: Create a file that, on close, is read-only and has its hash added to the audit log.
+ * Design: Overrides the File class
+ * Conditions: Always.
```

Comment:
`Conditions: Always`? I assume it means "when is this used," since other values include "during tabulation". Since "Always" is used by Logger, I assume it was appropriate to use here too -- i.e. no matter what you do (tabulate, log, launch gui, or do something on CLI), this will be used.

Reply:
Is this from some certification requirements?